gneiss/regression/_ols.py

```diff
@@ -7,13 +7,28 @@
 # ----------------------------------------------------------------------------
 from decimal import Decimal
 from collections import OrderedDict
-
+import numpy as np
 import pandas as pd
 from gneiss.regression._model import RegressionModel
 from ._regression import (_intersect_of_table_metadata_tree,
                           _to_balances)
 import statsmodels.formula.api as smf
 from statsmodels.iolib.summary2 import Summary
+from statsmodels.sandbox.tools.cross_val import LeaveOneOut
+from patsy import dmatrix
+
+
+def _fit_ols(y, x, **kwargs):
+    """ Perform the basic ols regression."""
+    exog_data = x
+    submodels = []
```

Comment:
Minor, and it depends on how big `len(y.columns)` is but this can be written as:
```python
submodels = [smf.OLS(endog=y[b], exog=x, **kwargs) for b in y.columns]
```

Appending to a list may take more than expected if the list is really long.